code/modules/reagents/chemistry/reagents/food.dm

```diff
@@ -660,20 +702,23 @@
 	description = "A starchy food paste made from boiled potatoes."
 	reagent_state = SOLID
 	color = "#D6D9C1"
+	taste_message = "potatoes"
 
 /datum/reagent/consumable/gravy
 	name = "Gravy"
 	id = "gravy"
 	description = "A savory sauce made from a simple meat-dripping roux and milk."
 	reagent_state = LIQUID
 	color = "#B4641B"
+	taste_message = "gravy"
 
 /datum/reagent/consumable/beff
 	name = "Beff"
 	id = "beff"
 	description = "An advanced blend of mechanically-recovered meat and textured synthesized protein product notable for its unusual crystalline grain when sliced."
 	reagent_state = SOLID
 	color = "#AC7E67"
+	taste_message = "meat; hopefully"
```

Comment:
The system lists things like so: `You can taste X, Y and Z.`. Adding full sentences or proper punctuation to the system will make the messages gibberish.

Though I do think this would do better with a hyphen.